jadx-core/src/test/java/jadx/tests/integration/conditions/TestConditions2.java

```diff
@@ -24,11 +29,12 @@ public void testComplexIf(String a, int b) {
 	}
 
 	@Test
+	@NotYetImplemented
 	public void test() {
 		ClassNode cls = getClassNode(TestCls.class);
 		String code = cls.getCode().toString();
 
-//		assertThat(code, containsString("return;"));
-//		assertThat(code, not(containsString("else")));
+		assertThat(code, containsString("return;"));
+		assertThat(code, not(containsString("else")));
```

Comment:
I think this is incorrect change, because original code checks only decompilation and recompilation without any manual code checks. But after change decompilation can fail but test will not show that.
I suggest just remove commented code, because checks is useless, and generated code is a same as original.
(NYI annotation also must be removed)

Reply:
I see, I thought it is a complex `return` that should be inline into one line.